Makefile.js

```diff
@@ -635,7 +635,27 @@ target.gensite = function(prereleaseVersion) {
         };
     }
 
-    // 4. Loop through all files in temporary directory
+    // 4. Find out which rules are recommended and/or fixable
+
+    const recommended = [];
+    const fixable = [];
+
+    find(path.join(process.cwd(), "/lib/rules/")).filter(fileType("js")).forEach(filename => {
+        const rule = require(filename);
```

Comment:
Would it be possible to use `require('.').linter.getRules()` to get all the rules, instead of looking through the filesystem?